services/common/src/redux/reducers/reportReducer.ts

```diff
@@ -38,6 +46,12 @@ const reportReducerObject = {
 export const getReports = (state) => state[REPORTS].reports;
 export const getReportsPageData = (state) => state[REPORTS].reportsPageData;
 export const getMineReports = (state) => state[REPORTS].mineReports;
+export const getMineReportById = (state, reportGuid) => {
```

Comment:
Is this function used to get the latest report or the first report ?

Reply:
it is meant to get the whole report, including all submissions.